PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc

```diff
@@ -66,6 +69,11 @@ PATElectronProducer::PATElectronProducer(const edm::ParameterSet & iConfig) :
   addGenMatch_(iConfig.getParameter<bool>( "addGenMatch" )),
   embedGenMatch_(addGenMatch_ ? iConfig.getParameter<bool>( "embedGenMatch" ) : false),
   embedRecHits_(iConfig.getParameter<bool>( "embedRecHits" )),
+  // for mini-iso
+  pcToken_(consumes<pat::PackedCandidateCollection>(iConfig.getParameter<edm::InputTag>("pfCandsForMiniIso"))),
```

Comment:
this call should be conditional on the value of computeMiniIso.
The consumes should not be called if computeMiniIso is false.
Please change here and for the electrons.